R/main.R

```diff
@@ -20,6 +20,23 @@
 #' @return List with information on optimal portfolio. Includes investment
 #'  in every candidate, benefits, cost, and total capacity.
 #' @export
+#' @examples
+#' population <- 31.99 
+#' gdp_pc <- 6.71
+#' frac_high_risk <- 0.131
+#' loss2yr <- 0.269
+#' price <- 4
+#' portfolio <- portfolioPriceTaker(population=population, gdp_pc=gdp_pc,
+#'                    frac_high_risk=frac_high_risk, 
+#'                    loss2yr=loss2yr, price=price,
+#'                    candidateFile="extdata/vaccinesSummary.csv")
```

Comment:
This should be `system.file()` by default?

Reply:
The same problem as the last comment. This is made default as I use system.file() within the function.